Packs/XSOARmirroring/Integrations/XSOARmirroring/README.md

```diff
@@ -1,230 +1,470 @@
+    | Entry Categories | Which entries to retrieve from the XSOAR server. The available options are notes, comments (chats) and files (attachments). | False |
+    | Incoming Entry tags | Only entries with these tags are retrieved from the XSOAR server. If no tags are listed, no entries are retrieved. | False |
+    | Outgoing Entry Tags | Choose the tags to filter the entries you wish to send to the other XSOAR instance. If no tags are listed, no entries will be sent. | False |
+    | Incident Mirroring Direction |  | False |
+    | Disable fetching for incidents came from this integration | Enable this option to disable mirroring of incidents that came from the integration of XSOAR Mirroring. This adds \`-sourceBrand:“XSOAR Mirroring”\` to your query. | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | Debug mode (will print debug logs to info) |  | False |
+    | Mirror Playbook ID | A parameter integration that removes the playbookId field from incoming incidents. Note: when set to true \(default\), the instance will attempt to run a playbook according to the incoming ID. When set to false, the instance will run the default playbook for the incident type \(if configured locally\). | False |
+
+4. To set up the mirroring, enable *Fetching incidents* in your instance configuration.
+5. In the *Incident Mirroring Direction* integration parameter, select in which direction the incidents should be mirrored:
+
+    | **Option** | **Description** |
+    | --- | --- |
+    | None | Turns off incident mirroring. |
+    | Incoming | Any changes in XSOAR Mirroring events (mirroring incoming fields) will be reflected in Cortex XSOAR incidents. |
+    | Outgoing | Any changes in Cortex XSOAR incidents will be reflected in XSOAR Mirroring events (outgoing mirrored fields). |
+    | Incoming And Outgoing | Changes in Cortex XSOAR incidents and XSOAR Mirroring events will be reflected in both directions. 
+   Newly fetched incidents will be mirrored in the chosen direction. However, this selection does not affect existing incidents.
+
+6. Click **Test** to ensure that you can communicate with the Cortex XSOAR tenant.
 
 
 ## Important notes:
 - In order to mirror custom fields, you need to create an incoming mapper for the integration and explicitly specify them in it.
 - In order to mirror custom fields in both directions, the custom fields in both XSOAR instances must have the same cli name.
+- Mirrored incidents include the playbook ID. The receiving side will attempt to run a playbook with a matching ID, if one exists locally. To have the machine run the default playbook for the mirrored incident, set the `Mirror Playbook ID` to `false`. Otherwise (default), the machine will attempt to run a playbook whose id matches the `playbookId` field in the mirrored incident.
```

Comment:
```suggestion
- Mirrored incidents include the playbook ID. The receiving side will attempt to run a playbook with a matching ID, if one exists locally. To have the machine run the default playbook for the mirrored incident, set the `Mirror Playbook ID` to `false`. Otherwise (default), the machine will attempt to run a playbook whose ID matches the `playbookId` field in the mirrored incident.
```